ubuntu-advantage

```diff
@@ -2,13 +2,14 @@
 
 SCRIPTNAME=$(basename "$0")
 
-SERIES=`lsb_release -cs`
+SERIES=$(lsb_release -cs)
```

Comment:
Keeping this one, there are no other backticks used in the script (and there are other instances of `$()`, so I'm actually unifying things).  Also, `shellcheck` doesn't like it:
```
dpb@helo:ubuntu-advantage-script[(25-ca-certs)]$ shellcheck -s sh ubuntu-advantage

In ubuntu-advantage line 5:
SERIES=`lsb_release -cs`
       ^-- SC2006: Use $(..) instead of legacy `..`.

```

I want to add shellcheck as a make target, but... another branch I think.